### PR TITLE
codemirror: Added missing options of setBookmark.

### PR DESCRIPTION
**Pull request by fefrei**
This is a minor PR, just adding two missing options of [setBookmark](https://codemirror.net/doc/manual.html#setBookmark) for `codemirror`.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://codemirror.net/doc/manual.html#setBookmark
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## codemirror ([<kbd>unpkg</kbd>](https://unpkg.com/browse/codemirror@latest/))
was missing the following properties:
1. optionHandlers
1. inputStyles
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 2fdd6398b31c9e6ee364b1434736735606ea5ec3
</p>

**Comment by fefrei** [comment]
(Just as a comment: I didn't add a test because the whole method to which I added options is untested, it seems – I just wanted to add the missing options.)

**Comment by fefrei** [comment]
I added more missing options, this time of the main `EditorConfiguration`. I noticed that some existing options are typed `any`, but I didn't want to touch them.

File: types/codemirror/index.d.ts [hunk 1]
```diff
@@ -1079,6 +1088,15 @@ declare namespace CodeMirror {
         that might change its proper indentation (only works if the mode supports indentation). Default is true. */
         electricChars?: boolean;
 
+        /** A regular expression used to determine which characters should be replaced by a special placeholder. Mostly useful for non-printing special characters. The default is /[\u0000-\u001f\u007f-\u009f\u00ad\u061c\u200b-\u200f\u2028\u2029\ufeff\ufff9-\ufffc]/. */
+        specialChars?: RegExp;
+
+        /** A function that, given a special character identified by the specialChars option, produces a DOM node that is used to represent the character. By default, a red dot (•) is shown, with a title tooltip to indicate the character code. */
+        specialCharPlaceholder?: (char: string) => HTMLElement;
```
Comment (andrewbranch) [inline types/codemirror/index.d.ts:1095]: Should this be `Element` instead of `HTMLElement`?

**Review by andrewbranch — APPROVED** [review]

File: types/codemirror/index.d.ts (hunk 1 above)
Comment (fefrei) [reply types/codemirror/index.d.ts:1095]: Good question. I'm not entirely sure, but you are _probably_ right. I'll go change it.

Comment (fefrei) [reply types/codemirror/index.d.ts]: Hm. Looks like `react-codemirror` disagrees?

Comment (andrewbranch) [reply types/codemirror/index.d.ts:1095]: Where are you seeing that?

Comment (fefrei) [reply types/codemirror/index.d.ts:1095]: The build failed, with the log mentioning `codemirror-react`. I'm not actually sure where to find that again (I already pushed a revert commit).

Comment (andrewbranch) [reply types/codemirror/index.d.ts:1095]: Something weird is going on with CI not related to your changes. I think the change was fine. But we can merge it either way.

**Review by andrewbranch — APPROVED** [review]

File: types/codemirror/index.d.ts (hunk 1 above)
Comment (fefrei) [reply types/codemirror/index.d.ts:1095]: Will keep it like this for now – maybe my first PR in this project is not the best one to fight CI over a change I'm not sure about. Thanks for your help!

**Comment by fefrei** [comment]
@typescript-bot: Ready to merge

**Comment by fefrei** [comment]
Ready to merge

**Pull request merged**